libr/anal/esil.c

```diff
@@ -3584,27 +3584,26 @@ static int evalWord(RAnalEsil *esil, const char *ostr, const char **str) {
 	return 3;
 }
 
-static bool __stepOut(RAnalEsil *esil, const char *cmd) {
+static bool __stepOut(RAnalEsil *esil, const char *cmd, bool *inCmdStep) {
```

Comment:
change the callers accordingly. but maybe its better to just move this inCmdStep into RAnalEsil struct instead of adding another parameter